live_tests/modules/test_texter.py

```diff
@@ -48,6 +57,20 @@ def test_preg_update_hindi_flow(self):
                                                join_keyword=hindi_remind(),
                                                born_keyword=hindi_born())
 
+    def decode_nonenglish_messages(self, language, messages, textlocal_object):
+        if language is not "English":
+            for index, text_tuple in enumerate(messages):
+                list_message = list(text_tuple)
+                if list_message[0].startswith("@U"):
+                    list_message[0] = textlocal_object.response_unicode_encoder(text_tuple[0])
+                    messages[index] = tuple(list_message)
+
+        return messages
```

Comment:
^ I don't really get what this helper is doing. Will we need it in the actual system?